jwala-services/src/main/java/com/cerner/jwala/service/jvm/state/JvmStateReceiverAdapter.java

```diff
@@ -69,18 +70,73 @@ public void receive(final Message jGroupMsg) {
         }
     }
 
+    private JvmState getJvmState(final Map serverInfoMap) {
+
+        // first check if the key types are string
+        final Set keys = serverInfoMap.keySet();
+        if (keys.isEmpty()) {
+            return null;
+        }
+
+        final Object initialKey = keys.iterator().next();
+        if (initialKey instanceof String) {
```

Comment:
Ok, got it now. I'll change it use contains(). 

This will actually be more flexible if there are any future changes since jwala owns the tomcat-agent module now.